examples/lima/coder.yaml

```diff
@@ -63,7 +62,7 @@ provision:
       # SSH session around. We don't want users to have to manually delete ~/.lima/$VM/ssh.sock
       # so we're just instead going to modify the perms on the Docker socket.
       # See: https://github.com/lima-vm/lima/issues/528
-      chown ${LIMA_CIDATA_USER} /var/run/docker.sock
+      chown {{.User}} /var/run/docker.sock
```

Comment:
Is this yaml processed by Go templating? Or is this something else?

Reply:
Yep, `lima` formats it.